@here/harp-mapview/test/VisibleTileSetTest.ts

```diff
@@ -271,8 +371,136 @@ describe("VisibleTileSet", function() {
         }
     });
 
+    /**
```

Comment:
There is no tests that checks if tiles below non-covering data-sources are actually not removed